src/test/resources/test_data/overall_calculation/consecutive-sentences/crs-2283-ac5-5.json

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
I'd suggest using the `BookingAnonymiser` for all your test cases, it makes them much easier to read.